shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -762,22 +762,6 @@ - (void)onLocaleUpdated:(NSNotification*)notification {
   NSArray<NSString*>* preferredLocales = [NSLocale preferredLanguages];
   NSMutableArray<NSString*>* data = [[NSMutableArray new] autorelease];
 
-  // Force prepend the [NSLocale currentLocale] to the front of the list
```

Comment:
Fun fact: you can change the app's locale and language with launch args `-AppleLanguages` and  `-AppleLocale`.
https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/TestingYourInternationalApp/TestingYourInternationalApp.html#//apple_ref/doc/uid/10000171i-CH7-SW2

> Alternatively, add AppleLanguages and AppleLocale launch arguments using the scheme editor—for example, add -AppleLanguages "(de)" to specify the German language and -AppleLocale "fr_FR" to specify the France region.

Reply:
Thanks, Will be super useful for the next set of changes that I'll be making immediately after this one!